ACHDictionary.go

```diff
@@ -132,3 +137,13 @@ func (f *ACHDictionary) parseParticipant() error {
 	f.IndexParticipant[p.RoutingNumber] = p
 	return nil
 }
+
+// RoutingNumberSearch returns a FEDACH participant based on a Participant.RoutingNumber.  Routing Number validation
+// is only that it exists in IndexParticipant.  Expecting 9 digits, checksum needs to be included
+// ToDo: Should this remain exportable?
+func (f *ACHDictionary) RoutingNumberSearch(s string) *Participant {
```

Comment:
We could reuse the validation func from ach: https://github.com/moov-io/ach/blob/master/validators.go#L494